rpc/route.go

```diff
@@ -32,6 +38,23 @@ func (r *router) routeRemote(method string) bool {
 	return r.methods[method]
 }
 
+func (r *router) routeBlocked(method string) bool {
+	_, ok := r.blockedMethods[method]
+	return ok
+}
+
+// blockedMethods is a list of dangerous or having security implications JSON-RPC methods
+// that are not allowed to be called.
+var blockedMethods = [...]string{
```

Comment:
I think that router becomes more complicated and it's worth to refactor it, so instead of having `local/remote` flag we can have more choices. Should be easier to grasp and understand. But that can be done as a separate refactoring.